serve.go

```diff
@@ -221,7 +221,11 @@ func serve(args []string) error {
 	}
 
 	if *flRepoPath != "" {
-		r.PathPrefix("/repo/").Handler(http.StripPrefix("/repo/", http.FileServer(http.Dir(*flRepoPath))))
+		if _, err := os.Stat(*flRepoPath); err == nil {
```

Comment:
You might also want to explicitly check for https://golang.org/pkg/os/#IsNotExist